Open Judge System/Workers/OJS.Workers.ExecutionStrategies/NodeJsPreprocessExecuteAndCheckExecutionStrategy.cs

```diff
@@ -183,16 +183,18 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
             // In NodeJS there is no compilation
             result.IsCompiledSuccessfully = true;
 
+            var executor = new RestrictedProcessExecutor();
+
             // Preprocess the user submission
```

Comment:
Do we need all these comments on every action? Code is quite self explanatory. @KristianMariyanov ?